rest/models/types.go

```diff
@@ -15,6 +15,15 @@ const (
 	Crypto MarketType = "crypto"
 )
 
+// ContractType is the type of contract.
+type ContractType string
+
+const (
+	ContractTypeCall  ContractType = "call"
```

Comment:
I think `ContractCall` would more closely match the style of the other enums in this repo

Reply:
Changed